css-nav-1/Overview.bs

```diff
@@ -1392,6 +1250,79 @@ Note: the 'spatial-navigation-contain' property is <a>at-risk</a>.
 
 </div>
 
+<h3 id=css-property-spatialnavigationaction>
+Defining the spatial navigation behavior : the 'spatial-navigation-action' property</h3>
+
+<pre class='propdef'>
+Name: spatial-navigation-action
+Value: auto | focus | scroll
+Initial: auto
+Applies to: <a>spatial navigation focus container</a>
```

Comment:
@frivoal Which is the most correct description among:
(1) all element
(2) spatial navigation focus container
(3) scrollable spatial navigation focus container

(1) is possible, but specifying this property is meaningless for the element which isn't a spatial navigation container.

Reply:
You can refer to the following links in terms of the consideration above.
https://www.w3.org/TR/cssom-view-1/#smooth-scrolling
https://www.w3.org/TR/css-scroll-snap-1/#scroll-padding

Reply:
I think this property is meaningful for the scrollable element.
Other properties such as https://drafts.csswg.org/cssom-view/#smooth-scrolling uses 
`scrolling box` as its applied element, so I'll change to it.

Reply:
Great! seems no problem now. +1